test/prompt/test_prompt.py

```diff
@@ -0,0 +1,19 @@
+import os
+import sys
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "..")))
```

Comment:
I'm not sure it's needed, it works fine without it for me

Reply:
Hm... I get:
```
ImportError while importing test module '/workspaces/prr/test/prompt/test_prompt_config.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/conda/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
test/prompt/test_prompt_config.py:5: in <module>
    from prr.prompt.model_options import ModelOptions
E   ModuleNotFoundError: No module named 'prr'
```

Reply:
Let's keep it then, it doesn't hurt I guess :man_shrugging: